packages/svelte/tests/signals/test.ts

```diff
@@ -488,7 +488,7 @@ describe('signals', () => {
 			assert.equal(a?.deps?.length, 1);
 			assert.equal(s?.reactions?.length, 1);
 			destroy();
-			assert.equal(a?.deps, null);
+			assert.equal(a?.deps?.length, 1);
```

Comment:
The derived still gets GC'd so this check isn't all that important. I think it's best to remove this line.